### ISSUE TITLE
Getting Traceback from a QRZ lookup. 

### ISSUE THREAD

**Issue by mbridak** [mbridak/not1mm#210]
Not sure why yet. But it shouldn't be happening.

```python
Traceback (most recent call last):
  File "/home/mbridak/Nextcloud/dev/not1mm/not1mm/__main__.py", line 3238, in callsign_changed
    self.lookup_service.msg_from_main(cmd)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "/home/mbridak/Nextcloud/dev/not1mm/not1mm/lookupservice.py", line 65, in msg_from_main
    result = self.look_up.lookup(call)
  File "/home/mbridak/Nextcloud/dev/not1mm/not1mm/lib/lookup.py", line 212, in lookup
    return root.get("Callsign")
           ^^^^
UnboundLocalError: cannot access local variable 'root' where it is not associated with a value
```

**Comment by mbridak** [comment]
```python
    def lookup(self, call: str) -> dict:
        """
        Lookup a call on QRZ
        """
        logger.info("%s", call)
        if self.session:
            ...
            # insert brilliant code here.
        return root.get("Callsign")
```

If there is no session root never gets defined.

**Comment by mbridak** [comment]
fixed with PR #211
